base_station/gui/src/components/Cameras.vue

```diff
@@ -109,8 +109,9 @@
       return {
         dual_stream: false,
         pi_index_1: -1,
-        pi_index_2: -1
-      }
+        pi_index_2: -1,
```

Comment:
Are we able to rename for pi_index to something else that is less confusing for the future (since we no longer use pi cams)?